src/geocel/GeantGeoParams.cc

```diff
@@ -131,6 +133,98 @@ std::vector<Label> make_physical_vol_labels(G4VPhysicalVolume const& world,
         });
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Push back an ordered list of "skin" (boundary) surfaces.
+ */
+void append_skin_surfaces(GeantGeoParams const& geo,
+                          std::vector<G4LogicalSurface const*>& result)
+{
+    // Translate "skin" (boundary) surfaces
+    using G4Surface = G4LogicalSkinSurface;
+    std::map<VolumeId, G4Surface const*> temp;
+    auto const* table = G4Surface::GetSurfaceTable();
+    CELER_ASSERT(table);
+#if G4VERSION_NUMBER >= 1120
+    for (auto&& [lv, surf] : *table)
+#else
+    for (auto const* surf : *table)
+#endif
+    {
+#if G4VERSION_NUMBER < 1120
+        auto* lv = surf->GetLogicalVolume();
+#endif
+
+        {
+            CELER_ASSERT(lv);
+            auto vol_id = geo.geant_to_id(*lv);
+            CELER_ASSERT(vol_id);
+            auto iter_inserted = temp.insert({vol_id, surf});
+            CELER_ASSERT(iter_inserted.second);
+        }
```

Comment:
```suggestion
        CELER_ASSERT(lv);
        auto vol_id = geo.geant_to_id(*lv);
        CELER_ASSERT(vol_id);
        auto iter_inserted = temp.insert({vol_id, surf});
        CELER_ASSERT(iter_inserted.second);
```
Redundant inner block